app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -35,7 +43,7 @@ data class Event(
       ticket =
           EventTicket(
               name = map["ticket_name"] as String,
-              price = map["ticket_price"] as Double,
+              price = convertToDouble(map["ticket_price"]),
```

Comment:
This feels a bit strange to me. Your `convertToDouble` function takes literally *anything* and tries to convert it into a `Double`. Why do you need a special function to convert to double when you do it like this `longitude = map["location_lng"] as Double` for the longitude?

Reply:
Note: if you **really** something similar to `convertToDouble`, I'd suggest using Kotlin's **safe casts**, i.e.,

```kotlin
val p = dataFromFirebase["my_property"] as? Type1 // Notice the "?"
```

`as? Type1` attempts to cast the value to `Type1`. However, instead of throwing a `ClassCastException` if the cast fails, `as?` returns `null`

Note that the value `p` is of type `Type1?` (nullable `Type1`)

Reply:
This code was introduced specifically for UI testing(I needed it to run the app) and was actually part of PR #270 , which was later merged. 
The issue, discussed with @Viccha18, arises when natural numbers are input into the database; they are recognized as Long types, we opted for this method since we didn't want values like 0 being cast to null. For latitude and longitude fields, we do not input natural numbers, so direct casting as Double suffices there. This convertToDouble function is a temporary measure and will be deprecated once the 'create Event' feature is implemented, ensuring all relevant values are explicitly managed as Double.